src/components/ReticleCanvas/ReticleCanvas.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useEffect, useRef } from "react";
+import Tilt from "react-parallax-tilt";
+import { getRelativeDiePositions } from "../../utils/calculations";
+import {defaultFieldWidth, defaultFieldHeight} from "../../config";
+
+type Props = {
+	mmToPxScale: number;
+	dieWidth: number;
+	dieHeight: number;
+	scribeHoriz: number;
+	scribeVert: number;
+	halfField: boolean;
+	showReticleBackground: boolean;
+};
+
+export function ReticleCanvas(props: Props) {
+	const canvasEl = useRef<HTMLCanvasElement>(null);
+
+	useEffect(() => {
+		if (!canvasEl.current) {
+			return;
+		}
+
+		const context = canvasEl.current.getContext("2d");
+
+		if (!context) {
+			return;
+		}
+
+		// Clear the canvases before drawing new die map
+		context.clearRect(0, 0, canvasEl.current.width, canvasEl.current.height);
+
+		// If “half-field” checkbox is checked, AKA High-NA, the reticle size doesn't
+		// change, but die appear double width on the reticle. In the real-world process,
+		// anamorphic mirrors are used to demagnify die by 8x horizontally and 4x
+		// vertically.
+		const { dieHeight } = props;
+		const dieWidth = props.halfField ? props.dieWidth * 2 : props.dieWidth;
```

Comment:
@jeff-the-k this is where I have implemented the die dimensions for the reticle visualization when Half Field Exposures are enabled (8x width, 4x height)